.gitignore

```diff
@@ -39,7 +39,6 @@ playground.xcworkspace
 # Add this line if you want to avoid checking in source code from Swift Package Manager dependencies.
 # Packages/
 # Package.pins
-Package.resolved
```

Comment:
I’m not clear on the workflow consequences of this change.

Reply:
Not sure why it would matter?

Reply:
My expectation would be that before this change, people would get the latest compatible-according-to-semver versions of dependencies, but with them, we’re pinned to specific revisions. That implies that someone™ should periodically be updating the pinned revisions.

Reply:
I assumed that would only be if you're building _this_ project, not when using mobius as a dependency of your own project. AFAIK it only matters for the top-level package. I could verify that if you have a concern there.

Reply:
That pretty much has to be true, but I don’t particularly like growing obsolescence in CI (and my local builds in Xcode 11.4) either

Reply:
I get that. I think it's good to pin _our_ dependencies in this repo to avoid weird discrepancies (builds fine on your machine but not mine).